### PR TITLE
feat: Error message for duplicate import of watch wallet

### PR DESCRIPTION
**Pull request by devchenyan**
issue: https://github.com/Magickbase/neuron-public-issues/issues/380

https://github.com/user-attachments/assets/497fa2a5-276e-42dd-bd52-133435afcf3f

### CODE REVIEW

**Review by yanguoyu — APPROVED** [review]

File: packages/neuron-ui/src/containers/Main/hooks.ts [hunk 1]
```diff
@@ -1,5 +1,6 @@
 import { useEffect, useCallback, useState } from 'react'
 import { useLocation, NavigateFunction, useNavigate } from 'react-router-dom'
+import { TFunction } from 'i18next'
```
Comment (Keith-CY) [inline packages/neuron-ui/src/containers/Main/hooks.ts:3]: `TFunction` can be imported as a `type`

File: packages/neuron-ui/src/locales/en.json [hunk 2]
```diff
@@ -1234,6 +1234,10 @@
         "tip": "Due to insufficient disk space, synchronization has been stopped. <br /> Please allocate more disk space or migrate the data to another disk.",
         "continue-sync": "Continue Sync",
         "migrate-data": "Migrate Data"
+      },
+      "import-exist-xpubkey-dialog": {
+        "replace-tip": "The watch wallet has been imported before, would you replace it?",
```
Comment (Keith-CY) [inline packages/neuron-ui/src/locales/en.json:1239]: `wallet wallet` should be `watch-only wallet`

Ref: https://river.com/learn/terms/w/watch-only-bitcoin-wallet/

File: packages/neuron-ui/src/locales/en.json [hunk 3]
```diff
@@ -1234,6 +1234,10 @@
...
+        "delete-tip": "The same original wallet existed, if you want to continue importing please delete it."
```
Comment (Keith-CY) [inline packages/neuron-ui/src/locales/en.json:1240]: The expression `if you want to continue importing please delete it.` is a bit weird. How about "If you want to continue with the import, please delete it."(from chatgpt)

**Review by Keith-CY — APPROVED** [review]

**Comment by Danie0918** [comment]
@Keith-CY Please check and merge.

**Review by Keith-CY — APPROVED** [review]

**Review by silySuper — APPROVED** [review]

**Pull request merged**